NAMESPACE

```diff
@@ -45,4 +45,4 @@ import(checkmate)
 importFrom(data.table,".I")
```

Comment:
side note: do we have to import these additionally as we import `data.table` (see: Imports in DESCRIPTION)